BDArmory/Modules/MissileFire.cs

```diff
@@ -2334,9 +2356,9 @@ IEnumerator LegacyCMRoutine()
             }
 
             if (BDArmorySettings.DRAW_DEBUG_LABELS && distance < 1000f) Debug.Log("[BDArmory.MissileFire]: Legacy missile warning for " + vessel.vesselName + " at distance " + distance.ToString("0.0") + "m from " + ml.shortName);
-            missileIsIncoming = true;
-            incomingMissileLastDetected = Time.time;
-            incomingMissileDistance = distance;
+            //missileIsIncoming = true;
```

Comment:
Repurposing the defunct legacyTargetVessel in Missilebase/Launcher caused some similarly obsolete target warning code to re-activate, so disabling it again.